src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-left: 50px;
+  margin-top: 5px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
```

Comment:
Let's create separate file for the styles for `nav` BEM block and don't forget about nesting